assets/js/angelleye-in-context-checkout.js

```diff
@@ -471,8 +471,15 @@ jQuery(function ($) {
                                 request_from: 'JSv4'
                             };
                             paypal.request.post(data.returnUrl, params).then(function (res) {
-                                data.returnUrl = res.url;
-                                actions.redirect();
+                                if( angelleye_in_content_param.is_cartflow === "no" ) {
+                                    data.returnUrl = res.url;
+                                    actions.redirect(); //data.returnUrl = res.url;
+                                    actions.redirect();
```

Comment:
@kcppdevelopers here actions.redirect() line is written twice. Low priority fix.